Makefile

```diff
@@ -714,4 +714,6 @@ help:
 
 migrations-extract:
 	@echo Listing migration files
-	find db/migrations -maxdepth 2 -mindepth 2 | sort > db/migrations/migration.list
+	rm -f db/migrations/migrations.list
```

Comment:
Why is it deleted here when it's overwritten in the next line anyway?

Reply:
Ah, yeah need to remove that line. Thanks!